### ISSUE TITLE
Why the loss has only loss_ce of Upernet?

### ISSUE THREAD

**Issue by zhihou7** [open-mmlab/mmsegmentation#1166]
Thanks for your error report and we appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**
Hi,

I trained Upernet as follow,

    ./tools/dist_train.sh configs/vit/upernet_deit-b16_512x512_80k_ade20k.py 8

The log is,

       mmseg - INFO - Iter [300/80000]       lr: 1.192e-05, eta: 17:47:55, time: 0.683, data_time: 0.004, memory: 8125, decode.loss_c
e: 3.4515, decode.acc_seg: 22.1028, aux.loss_ce: 1.5121, aux.acc_seg: 20.2416, loss: 4.9637

However, the provided log file in https://download.openmmlab.com/mmsegmentation/v0.5/vit/upernet_deit-b16_512x512_80k_ade20k/20210624_130529.log.json is,

       {"mode": "train", "epoch": 1, "iter": 300, "lr": 1e-05, "memory": 13071, "data_time": 0.00795, "decode.loss_seg": 2.81133, "decode.acc_seg": 40.63443, "aux.loss_seg": 1.42657, "aux.acc_seg": 38.01873, "loss": 4.23789, "time": 0.40944}

In the provided log file, there are no loss_ce, but loss_seg. Meanwhile, I find the acc_seg is also much worse acc_seg in provided log file.  Meanwhile, the result is much worse the reported (27 vs 45).

Is there are any  bugs imported in recent mmsegmentation version?

**Reproduction**

1. What command or script did you run?

    ```none
      ./tools/dist_train.sh configs/vit/upernet_deit-b16_512x512_80k_ade20k.py 8
    ```

2. Did you make any modifications on the code or config? Did you understand what you have modified? No
3. What dataset did you use? ADE150

**Environment**

1. Please run `python mmseg/utils/collect_env.py` to collect necessary environment information and paste it here.

sys.platform: linux
Python: 3.8.10 | packaged by conda-forge | (default, May 11 2021, 07:01:05) [GCC 9.3.0]
CUDA available: True
GPU 0,1,2,3,4,5,6,7: A100-SXM4-40GB
CUDA_HOME: /usr/local/cuda
NVCC: Build cuda_11.3.r11.3/compiler.29920130_0
GCC: gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
PyTorch: 1.9.0a0+c3d40fd
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2019.0.5 Product Build 20190808 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.1.2 (Git Hash N/A)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.3
  - NVCC architecture flags: -gencode;arch=compute_52,code=sm_52;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_86,code=compute_86
  - CuDNN 8.2.1
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.3, CUDNN_VERSION=8.2.1, CXX_COMPILER=/usr/bin/c++, CXX_FLAGS= -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.9.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=ON, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON,

TorchVision: 0.10.0a0
OpenCV: 4.4.0
MMCV: 1.4.0
MMCV Compiler: GCC 9.3
MMCV CUDA Compiler: 11.3
MMSegmentation: 0.19.0+5049651

**Error traceback**

If applicable, paste the error trackback here.

```none
A placeholder for trackback.
```

**Comment by MengzhangLI** [comment]
(1) The model using `UPerNet` all use cross entropy loss, you can find it in their corresponding config files.

(2) The phenomenon you noticed because the log you read here is from some experiments when the `loss_name` is not added. Please see this pr:https://github.com/open-mmlab/mmsegmentation/pull/818. 
https://github.com/open-mmlab/mmsegmentation/blob/fa8c93e78c1679eee558ced5047cc280adfa1c1d/mmseg/models/losses/cross_entropy_loss.py#L206-L218

After this pr is merged, `loss_ce` would be added in log.
